tests/test_unit/test_cdf_tk/test_resource_classes/test_migration.py

```diff
@@ -0,0 +1,99 @@
+from collections.abc import Iterable
+from pathlib import Path
+
+import pytest
+
+from cognite_toolkit._cdf_tk.resource_classes.migration import ResourceViewMappingYAML
+from cognite_toolkit._cdf_tk.tk_warnings.fileread import ResourceFormatWarning
+from cognite_toolkit._cdf_tk.validation import validate_resource_yaml_pydantic
+
+
+def invalid_mapping_test_cases() -> Iterable:
+    yield pytest.param(
+        {"viewId": {"space": "cdf_cdm", "externalId": "CogniteAsset", "version": "v1"}},
+        {"Missing required field: 'propertyMapping'", "Missing required field: 'resourceType'"},
+        id="Missing required field: externalId",
```

Comment:
Why does the id say that `externalId` is missing when the expected error messages mention propertyMapping and resourceType?